src/mlpack/tests/bayesian_ridge_test.cpp

```diff
@@ -0,0 +1,145 @@
+/**
+ * @file bayesian_ridge_test.cpp
+ * @author Clement Mercier
+ *
+ * Test for BayesianRidge.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+
+#include <mlpack/core/data/load.hpp>
+#include <mlpack/methods/bayesian_ridge/bayesian_ridge.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+using namespace mlpack::regression;
+using namespace mlpack::data;
+
+BOOST_AUTO_TEST_SUITE(BayesianRidgeTest);
+
+void GenerateProblem(arma::mat& X,
+                     arma::rowvec& y,
+                     size_t nPoints,
+                     size_t nDims,
+                     float sigma = 0.0)
+{
+  arma::arma_rng::set_seed(4);
```

Comment:
I think this is an artifact from a debug session, for the test cases we avoid setting a fixed seed. Testing on different seeds often reveals unknown issues.